labelbox/schema/dataset.py

```diff
@@ -750,3 +755,35 @@ def _export(
         res = res[mutation_name]
         task_id = res["taskId"]
         return Task.get_task(self.client, task_id)
+
+    def upsert_data_rows(self, items: list[DataRowUpsertItem]) -> "Task":
```

Comment:
For now, let's also enforce this limit - https://github.com/Labelbox/labelbox-python/blob/develop/labelbox/schema/dataset.py#L34.

This matches the expectation for imports, in the future I can update the backend to remove this limitation.